content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,77 @@
+
+## Traffic management
+
+- **Added** [automatic determination](/docs/ops/traffic-management/protocol-selection/) of protocol types. You no longer need to explicitly prefix service ports with their protocol type.
+- **Added** a mode to the Gateway API for mutual TLS operation.
+- **Fixed** issues present when a service communicates over the network first in permissive mutual TLS mode for protocols like MySQL and MongoDB.
+- **Improved** Envoy proxy readiness checks. They now check Envoy's readiness status.
+- **Improved** container ports are no longer required in the pod spec. All ports are [captured by default](/faq/traffic-management/#controlling-inbound-ports).
+- **Improved** the `EnvoyFilter` API. You can now add or update all configurations.
+- **Improved** the Redis load balancer to now default to [`MAGLEV`](https://www.envoyproxy.io/docs/envoy/v1.6.0/intro/arch_overview/load_balancing#maglev) when using the Redis proxy.
+- **Improved** load balancing to direct traffic to the [same region and zone](/faq/traffic-management/#controlling-inbound-ports) by default.
+- **Improved** Pilot by reducing CPU utilization. The reduction approaches 90% depending on the specific deployment.
+- **Improved** the `ServiceEntry` API to allow for the same hostname in different namespaces.
+- **Improved** the [Sidecar API](/docs/reference/config/networking/v1alpha3/sidecar/#OutboundTrafficPolicy) to customize the `OutboundTrafficPolicy` policy.
+
+## Security
+
+- **Added** trust domain validation for services using mutual TLS. By default, the server only authenticates the requests from the same trust domain.
+- **Added** [labels](/docs/concepts/security/#how-citadel-determines-whether-to-create-service-account-secrets) to control service account secret generation by namespace.
+- **Added** SDS support to deliver the private key and certificates to each Istio control plane service.
+- **Added** support for [introspection](/docs/ops/troubleshooting/controlz/) to Citadel.
+- **Added** metrics to the `/metrics` endpoint of Citadel Agent on port 15014 to monitor the SDS service.
+- **Added** diagnostics to the Citadel Agent using the `/debug/sds/workload` and `/debug/sds/gateway` on port 8080.
+- **Improved** the ingress gateway to [load the trusted CA certificate from a separate secret](/docs/tasks/traffic-management/ingress/secure-ingress-sds/#configure-a-mutual-tls-ingress-gateway) when using SDS.
+- **Improved** SDS security by enforcing the usage of [Kubernetes Trustworthy JWTs](/blog/2019/trustworthy-jwt-sds).
+- **Improved** Citadel Agent logs by unifying the logging pattern.
+- **Removed** support for Istio SDS when using [Kubernetes versions earlier than 1.13](/blog/2019/trustworthy-jwt-sds).
+- **Removed** integration with Vault CA temporarily. SDS requirements caused the temporary removal but we will reintroduce Vault CA integration in a future release.
```

Comment:
I think this is only not working with our Vault toy example. I assume if a user deploys Vault in such as way that it can authenticate the SDS JWT (same apiserver with the workloads, or it does some kind of syncing/federating service accounts between projects/clusters), then this is fine?

Reply:
Let's fully verify that the Vault works with the new Trustworthy JWTs. We don't want to claim it's supported before we fully verify it's working :)